Header header
float64[] probs
int64[] inds
float64[] colors
